src/pages/Home.jsx

```diff
@@ -0,0 +1,5 @@
+const Home = () => {
+  return;
+};
+
+export default Home;
```

Comment:
This component is strange, please remove it if it is not being used, or return `null` if a component shouldn't return anything